app/src/main/java/acr/browser/lightning/reading/ReadingActivity.kt

```diff
@@ -115,7 +111,8 @@ class ReadingActivity : ThemedActivity(), TextToSpeech.OnInitListener {
         iTtsEngine = TextToSpeech(this, this)
 
         setContentView(R.layout.reading_view)
-        ButterKnife.bind(this)
+        mTitle = findViewById(R.id.textViewTitle)
+        mBody = findViewById(R.id.textViewBody)
```

Comment:
Could use actual binding instead. No big deal though.

Reply:
I can change it, no problem.